README.md

```diff
@@ -41,29 +45,40 @@ An example of a cycle in project dependenies is:
 For example, after running `depstat graph`, an SVG can be created using:
 `twopi -Tsvg -o dag.svg graph.dot`
 
+By default, the graph would be created around the main module (first module in the `go mod graph` output), but you can choose to create a graph around a particular dependency using the `--dep` flag.
+
 ### list
 
 `depstat list` shows a sorted list of all project dependencies. These include both direct and transitive dependencies.
 
-1. Direct dependencies: Dependencies that are directly used in the code of the project. These do not include standard go packages like `fmt`, etc.
+1. Direct dependencies: Dependencies that are directly used in the code of the project. These do not include standard go packages like `fmt`, etc. These are dependencies that appear on the right side of the main module in the `go mod graph` output.
 
-2. Transitive dependencies: These are dependencies that get imported because they are needed by some direct dependency of the project.
+2. Transitive dependencies: These are dependencies that get imported because they are needed by some direct dependency of the project. These are dependencies that appear on the right side of a dependency that isn't the main module in the `go mod graph` output.
 
 ### stats
 
 `depstat stats` will provide the following metrics about the dependencies of the project:
 
-1. Total Dependencies: Total number of dependencies of the project. This is the sum of direct and transitive dependencies.
+1. Direct Dependencies: Total number of dependencies required by the [main module(s)](#main-module) directly.
+
+2. Transitive Dependencies: Total number of transitive dependencies (dependencies which are further needed by direct dependencies of the project).
 
-2. Max Depth of Dependencies: Number of dependencies in the longest dependency chain present in the project.
+3. Total Dependencies: Total number of dependencies of the [main module(s)](#main-module).
 
-3. Transitive Dependencies: Total number of transitive dependencies.
+4. Max Depth of Dependencies: Length of the longest chain starting from the first [main module](#main-module); defaults to length from the first module encountered in "go mod graph" output.
 
 - The `--json` flag gives this output in a JSON format.
 - `--verbose` mode will help provide you with the list of all the dependencies and will also print the longest dependency chain.
 
+#### main module
+By default, the first module encountered in "go mod graph" output is treated as the main module by `depstat`. Depstat uses this main module to determine the direct and transitive dependencies. This behavior can be changed by specifying the main module manually using the `--mainModules` flag with the stats command. The flag takes a list of modules names, for example:
```

Comment:
tiny nit: The `--mainModules` flag is mentioned here but the `-m` flag is used below. Maybe mention the `-m` flag in the text description as well?

Reply:
Replaced it in the example to avoid more text :)